src/test/java/com/jcabi/github/mock/MkNotificationsTest.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2013-2015, jcabi.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.github.mock;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test case for {@link MkNotifications}.
+ *
+ * @author Piotr Pradzynski (prondzyn@gmail.com)
+ * @version $Id$
+ * @todo #952 Create a test fetchNonEmptyListOfNotifications and implement
+ *  iterate() operation in MkNotifications.
+ */
+public final class MkNotificationsTest {
+
+    /**
+     * MkNotifications can fetch empty list of Notification.
+     * @throws Exception if some problem inside
+     */
+    @Test
+    public void fetchEmptyListOfNotifications() throws Exception {
+        MatcherAssert.assertThat(
+            new MkNotifications().iterate(),
+            Matchers.emptyIterable()
+        );
+    }
+
+    /**
+     * MkNotifications can get single Notification.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void fetchSingleNotification() throws Exception {
```

Comment:
@prondzyn why we need to check this case?
I think check for empty and non empty collection will be enough